package.json

```diff
@@ -45,7 +45,7 @@
     "@sphereon/react-native-argon2": "2.0.9",
     "@sphereon/ssi-sdk-contact-manager": "0.9.1-next.103",
     "@sphereon/ssi-sdk-core": "0.9.1-next.103",
-    "@sphereon/ssi-sdk-data-store": "0.9.1-next.103",
+    "@sphereon/ssi-sdk-data-store": "0.9.1-next.107",
```

Comment:
need other ssi versions updated as well

Reply:
updated all the ssi-sdk versions to `0.9.1-next.107`